frontend/cache-handler.js

```diff
@@ -0,0 +1,34 @@
+const { IncrementalCache } = require("@neshca/cache-handler")
```

Comment:
If this has to be a js file, can we at least get a `// @ts-check` annotation?

Reply:
Done